static/app/routes.tsx

```diff
@@ -1193,6 +1193,15 @@ function routes() {
               }
               component={errorHandler(LazyLoad)}
             />
+            <Route
+              path="manage/"
```

Comment:
Long term will the 'manage' route be moved up to `/dashboards`? Or will that route continue to be your 'last used' dashboard?